requirements.txt

```diff
@@ -45,3 +45,10 @@ g2pkk>=0.1.1
 bangla==0.0.2
 bnnumerizer
 bnunicodenormalizer==0.1.1
+
+#deps for tortoise
+gdown
```

Comment:
if we load models from github releases we don't need it.

Reply:
gdown we can remove as i have remove bigvgan.